### PR TITLE
article added 'Edit your YAML like a boss'

### PR DESCRIPTION
**Pull request by bloodrabbit**

### CODE REVIEW

File: source/2016-02-12-edit-your-yaml-files-like-a-boss.html.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Edit your YAML like a boss
+date: 2016-02-12 13:24 UTC
+tags: features
+authors: Daniel Balla|danielballa@bitrise.io
+---
+
+![YAML editor](yaml_editor.png)
+
+Do you fancy editing your `bitrise.yml` sometimes, beside using the Workflow Editor? We have great news for you then, because we pimped the hell out of the .yml editor, using the great [UI.Ace Angular directive](https://github.com/angular-ui/ui-ace).
```
Comment (viktorbenei) [inline source/2016-02-12-edit-your-yaml-files-like-a-boss.html.markdown]: ACE editor

File: source/2016-02-12-edit-your-yaml-files-like-a-boss.html.markdown [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+As per every proper code editor, there is now proper **syntax highlighting** with the beloved Monokai color theme. Sublime users will recognize the familiar touch 😉.
```
Comment (viktorbenei) [inline source/2016-02-12-edit-your-yaml-files-like-a-boss.html.markdown]: proper proper?

Comment (viktorbenei) [inline source/2016-02-12-edit-your-yaml-files-like-a-boss.html.markdown]: ~~color~~

**Comment by bloodrabbit** [comment]
fixed!

**Pull request merged**